lib/net/winrm/connection.rb

```diff
@@ -0,0 +1,289 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'winrm'
+require 'winrm/wsmv/write_stdin'
+
+module Net
+  module MsfWinRM
+    class RexWinRMConnection < WinRM::Connection
+      class MessageFactory < WinRM::PSRP::MessageFactory
```

Comment:
Just confirming, is this class needed? It doesn't look like we're dealing with the PSRP protocol here

Reply:
Thanks - deleted.